release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -774,6 +774,17 @@ Providing a `hugepages` property into the `MachineSet` resource is now possible.
 
 For any other changes in `/etc/containers/registries.conf` files, the Machine Config Operator will default to draining nodes to apply changes. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1943315[*BZ#1943315*].
 
+[id="ocp-4-8-machine-management"]
+=== Machine management
+
+[id="ocp-4-8-poison-pill-operator]
+==== Reboot unhealthy nodes with the Poison Pill Operator
+
+You can use the Poison Pill Operator to automatically reboot unhealthy nodes. This minimizes downtime for stateful applications and ReadWriteOnce (RWO) volumes, and restores compute capacity if transient failures occur.
```

Comment:
I would highlight the fact that the node is rebooting itself (unlike eg. metal3)
And that no specialised hardware is required (unlike eg. metal3)
Clusters that do have such hardware should probably use it instead